logos/logos-orchestrator/tests/unit/sdi/test_scheduler_view.py

```
@@ -134,16 +133,14 @@ def test_scheduler_view_loaded_vllm_and_sleeping_ollama(monkeypatch):
     vllm_lane = next(l for l in view.lanes if l.lane_id == "vllm-1")
-    assert vllm_lane.is_vllm is True
     assert vllm_lane.requests_running == 2.0
     assert vllm_lane.gpu_cache_usage_percent == 45.0
     assert vllm_lane.gpu_memory_utilization == 0.7
     assert vllm_lane.tensor_parallel_size == 2
 
-    ollama_lane = next(l for l in view.lanes if l.lane_id == "ollama-1")
-    assert ollama_lane.is_vllm is False
-    assert ollama_lane.requests_running == 0.0  # Ollama uses active_requests for requests_running
-    assert ollama_lane.gpu_cache_usage_percent is None
+    sleeping_lane = next(l for l in view.lanes if l.lane_id == "sleeping-1")
```

Ruff (0.16.2)

[error] Ambiguous variable name: `l`

(E741)